web/package/agama-web-ui.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Jun 18 08:42:49 UTC 2025 - David Diaz <dgonzalez@suse.com>
+
+- Improve multilingual font support by fixing the font-face rule for
+  Chinese, adding Noto Sans Georgian, and making bundled fonts
+  available for the system via a new config in the live ISO
```

Comment:
This change could be also mentioned in the [live/src/agama-installer.changes](https://github.com/agama-project/agama/blob/master/live/src/agama-installer.changes) file.

Reply:
Right. I will adapt it